jot/UIColor+Jot.m

```diff
@@ -13,28 +13,28 @@
 
 @implementation UIColor (Jot)
 
-+ (UIColor *)jotBlack {
-    return UIColorFromRGB(0x4A4A4A);
++ (UIColor *)jotNavy {
```

Comment:
We could probably call these colors `patreon___` (i.e. `patreonNavy`) so that we can tell which ones are our brand colors and not part of the `jot` library.